.config/gitpod/Dockerfile

```diff
@@ -38,23 +36,37 @@ RUN PYTHON_VERSION=$(cut -d '=' -f 2 /tmp/python_version.config) && \
         vim \
         wget \
     && python3 -m pip install --no-cache-dir --upgrade pip \
-    && python3 -m pip install --no-cache-dir virtualenv \
-    # Install Docker
-    && curl -fsSL https://download.docker.com/linux/ubuntu/gpg | gpg --dearmor -o /usr/share/keyrings/docker-archive-keyring.gpg \
+    && python3 -m pip install --no-cache-dir virtualenv
+
+# Install Docker
+FROM builder as docker-installer
+
+RUN curl -fsSL https://download.docker.com/linux/ubuntu/gpg | gpg --dearmor -o /usr/share/keyrings/docker-archive-keyring.gpg \
     && echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/docker-archive-keyring.gpg] https://download.docker.com/linux/ubuntu \
     $(lsb_release -cs) stable" | tee /etc/apt/sources.list.d/docker.list > /dev/null \
     && apt-get update \
-    && apt-get install -y --no-install-recommends docker-ce docker-ce-cli containerd.io \
-    # Install npm and ungit using the versions specified in the variables
-    && curl -fsSL https://deb.nodesource.com/setup_20.x |bash - \
+    && apt-get install -y --no-install-recommends docker-ce docker-ce-cli containerd.io
+
+# Install npm and ungit
+FROM docker-installer as npm-installer
+
+RUN curl -fsSL https://deb.nodesource.com/setup_20.x | bash - \
     && apt-get install  -y --no-install-recommends nodejs \
     && npm install -g npm@${NPM_VERSION} \
```

Comment:
SC2086 info: Double quote to prevent globbing and word splitting.: To prevent glob expansion and word splitting, you can add double quotes around variables in RUN commands. For example, `"npm@${NPM_VERSION}"` instead of npm@${NPM_VERSION}.

Reply:
@ThomasSanson 

I have just made the modification related to the SC2086 error, preventing globbing and word splitting. I added double quotes as recommended. However, when running the "make tests" command, the SC2086 error was still present. Therefore, I also added double quotes to the term "ungit@${UNGIT_VERSION}". After re-running the "make tests" command, no further errors were generated.

Nevertheless, I kindly request you to verify for security purposes.

You can find the modification in this commit: 94607ef9625e360875f22f7a766b3df8a2f71694